js/index.js

```diff
@@ -218,6 +230,15 @@ function initialExec() {
   const inputMessage = document.getElementById('survey-form-message-input');
   inputMessage.addEventListener('focus', eraseAlert);
 
+  const obj = JSON.parse(localStorage.getItem('array'));
+  inputName.value = obj.name;
+  inputEmail.value = obj.email;
+  inputMessage.value = obj.message;
+
+  inputName.addEventListener('input', changeInput);
```

Comment:
- [ ] According to the project requirements:  When the user changes the content of any input field in the form the data object is updated and saved to the local storage. In order to be saved to the local storage the data object is turned into a string using `JSON.stringify` or similar methods. Ie:

```
localStorage.setItem('contactFormInput', JSON.stringify(formData));
```

- In order to be used after retrieved from the local storage the saved string is turned back into an object using `JSON.parse` or similar methods.

```
let savedFormData = localStorage.getItem('contactFormInput');
savedFormData = JSON.parse(savedFormData);
```

- [ ] Make sure When the user loads the page, the code checks if there are any saved data in the local storage and if so the input fields are pre-filled with the data.

```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};
```

- Please address this 👍🏻